hapi/hedera-protobufs/services/basic_types.proto

```diff
@@ -1269,6 +1269,11 @@ enum HederaFunctionality {
      * Submit a node public tss encryption key as part of the Threshold Signature Scheme (TSS).
      */
     TssEncryptionKey = 99;
+
+    /**
+     * Submit a signature of a state snapshot gossiped to other nodes
```

Comment:
it's not technically a snapshot, it's a state root hash. A snapshot is how the state is represented on disk.